rx/src/main/java/co/infinum/goldfinger/rx/RxGoldfinger.java

```diff
@@ -17,19 +17,19 @@
 public interface RxGoldfinger {
```

Comment:
I know it's not the scope of this PR, but maybe we should start thinking of adding coroutines support to this lib.

Reply:
Maybe with the refactor to kotlin, shiny Goldfinger 3.0.0 :)